src/modelmanager.cpp

```diff
@@ -434,6 +434,14 @@ void ModelManager::retireModelsRemovedFromConfigFile(const std::set<std::string>
     }
 }
 
+void ModelManager::updateConfigurationWithoutConfigFile() {
```

Comment:
Actually I especially extracted this method to not have 2 different logics inside real model manager and test one. This may be useful for model control API as well.

Reply:
What I could do eventually would be to have this method protected in ModelManager & public in ModelManager used in test but I think this may be an overkill.

Reply:
Will move those methods as protected & expose them in ConstructorEnabledModelManager.